wait/errors.go

```diff
@@ -1,3 +1,4 @@
+//go:build !windows
```

Comment:
@hairyhenderson was this enforced by the script?

Reply:
@mdelapenya yeah, because I'm using Go 1.17 locally... It shouldn't hurt anything on older Go runtimes.